### PR TITLE
Update product template site

### PR DESCRIPTION
**Pull request by mattempty**
This pull request changes the base template to use the latest version of the Product Pages from GDS. Content has been mapped across and site is live (using the older url).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "product-page-example",
+  "name": "moj-online-product-page",
   "private": "true",
   "version": "1.0.0",
-  "description": "This is an example product page for use by GaaP products.",
+  "description": "Theis is the product page for MoJ Online.",
```
Comment (brenetic) [inline package.json:5]:
```suggestion
  "description": "This is the product page for MoJ Online.",
```

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+all: clean build build-docker run-docker
+
+docker-go-away: remove-docker-containers remove-docker-images
+
+build-and-deploy: clean build live-deploy
+
+build:
+	@echo "Build..."
+	# git pull
```
Comment (brenetic) [inline Makefile:11]:
```suggestion
	~# git pull~
```

File: config.rb [hunk 3]
```diff
@@ -28,20 +25,19 @@
...
 #   set :analytics, "'UA-XXXXXXXX-X'"
-#end
+end
 
 # Insert your Google Analytics ID below
-# configure :build do
-#   set :analytics, "'UA-XXXXXXXX-X'"
-# end
+configure :build do
+  set :analytics, "'UA-129572512-1'"
```
Comment (brenetic) [inline config.rb:35]:
```suggestion
  set :analytics, "'UA-XXXXXXXX-X'"
```

File: .gitignore [hunk 4]
```diff
@@ -2,4 +2,5 @@
 .idea/
 source/.DS_Store
 node_modules/
-build/
\ No newline at end of file
+build/
+Makefile
```
Comment (brenetic) [inline .gitignore:6]: If we are going to ignore it do we want to check it into git?

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+live-deploy:
+	@echo "Building Image..."
+	docker build -t formbuilder/formbuilder-product-page .
+	@echo "Tagging Image..."
+	docker tag formbuilder/formbuilder-product-page:latest 754256621582.dkr.ecr.eu-west-2.amazonaws.com/formbuilder/formbuilder-product-page:latest
+	@echo "Pushing Image..."
+	docker push 754256621582.dkr.ecr.eu-west-2.amazonaws.com/formbuilder/formbuilder-product-page:latest
+	@echo "Restarting pod..."
+	./scripts/restart
+
```
Comment (brenetic) [inline Makefile:50]: If we're adding the Makefile to the gitgnore then we shouldn't check it in

Comment (mattempty) [reply Makefile:50]: I didn't think I had. How do I uncheck it in?

File: .gitignore (hunk 4 above)
Comment (mattempty) [reply .gitignore:6]: Ignore.

File: config.rb [hunk 6]
```diff
@@ -28,20 +25,19 @@
...
 #   set :analytics, "'UA-XXXXXXXX-X'"
-#end
+end
 
 # Insert your Google Analytics ID below
-# configure :build do
-#   set :analytics, "'UA-XXXXXXXX-X'"
-# end
+configure :build do
+  set :analytics, "'UA-XXXXXXXX-X'"
```
Comment (brenetic) [inline config.rb:35]: Are we going to use these two things now? If no maybe leave them commented out

Comment (mattempty) [reply config.rb]: Updated.

**Pull request merged**